aiven_db_migrate/migrate/pgmigrate.py

```diff
@@ -852,6 +865,17 @@ def _check_databases(self):
                 else:
                     self.log.info("Database %r already exists in target", dbname)
 
+    def _check_database_size(self, max_size: float):
```

Comment:
That function too should have specific tests (it's mocked in the indirect tests of `test_migrate_checks.py`)

Reply:
It's mocked only to assert that it gets called. It should pass the original function as side effect to make sure that it gets called. But I'll add some more tests.